### PR TITLE
[SHIMGVW] Update German (de-DE) translation

### PR DESCRIPTION
**Pull request by Venca24**
## Purpose

Translates the new strings added in #6186 into German.

JIRA issue: none

### CODE REVIEW

File: dll/win32/shimgvw/lang/de-DE.rc [hunk 1]
```diff
@@ -16,8 +16,8 @@ BEGIN
     IDS_TOOLTIP_ZOOM_OUT "Verkleinern (-)"
     IDS_TOOLTIP_ROT_CLOCKW "Drehen im Uhrzeigersinn (Strg+K)"
     IDS_TOOLTIP_ROT_COUNCW "Drehen gegen den Uhrzeigersinn (Strg+L)"
-    IDS_TOOLTIP_ROT_CWSAVE "Rotate Clockwise and Save (Lossy)"
-    IDS_TOOLTIP_ROT_CCWSAVE "Rotate Counterclockwise and Save (Lossy)"
+    IDS_TOOLTIP_ROT_CWSAVE "Drehen im Uhrzeigersinn und speichern (verlustbehaftet)"
```
Comment (gonzoMD) [inline dll/win32/shimgvw/lang/de-DE.rc:19]: Im Uhrzeigersinn drehen und speichern (verlustbehaftet)

File: dll/win32/shimgvw/lang/de-DE.rc [hunk 2]
```diff
@@ -16,8 +16,8 @@ BEGIN
...
+    IDS_TOOLTIP_ROT_CCWSAVE "Drehen gegen den Uhrzeigersinn und speichern (verlustbehaftet)"
```
Comment (gonzoMD) [inline dll/win32/shimgvw/lang/de-DE.rc:20]: Gegen den Uhrzeigersinn drehen und speichern (verlustbehaftet)

**Review by gonzoMD — COMMENTED** [review]
I would change the sentences structure a bit.

File: dll/win32/shimgvw/lang/de-DE.rc [hunk 3]
```diff
@@ -16,8 +16,8 @@ BEGIN
     IDS_TOOLTIP_ZOOM_OUT "Verkleinern (-)"
     IDS_TOOLTIP_ROT_CLOCKW "Drehen im Uhrzeigersinn (Strg+K)"
     IDS_TOOLTIP_ROT_COUNCW "Drehen gegen den Uhrzeigersinn (Strg+L)"
-    IDS_TOOLTIP_ROT_CWSAVE "Rotate Clockwise and Save (Lossy)"
-    IDS_TOOLTIP_ROT_CCWSAVE "Rotate Counterclockwise and Save (Lossy)"
+    IDS_TOOLTIP_ROT_CWSAVE "Im Uhrzeigersinn drehen und speichern (verlustbehaftet)"
+    IDS_TOOLTIP_ROT_CCWSAVE "Gegen den Uhrzeigersinn drehen und speichern (verlustbehaftet)"
```
Comment (JoachimHenze) [inline dll/win32/shimgvw/lang/de-DE.rc:20]: Now the wording of IDS_TOOLTIP_ROT_CLOCKW does mismatch IDS_TOOLTIP_ROT_CWSAVE and
IDS_TOOLTIP_ROT_COUNCW mismatches IDS_TOOLTIP_ROT_CCWSAVE.
Imho that was better before gonzomds proposal. Or at least the Strings in line 17 and 18 should be reordered in the wording as well, to make that match again. Either always Drehen in the beginning, or always the Drehen as the 2nd term of that help-text. But should not be mixed.

**Review by gonzoMD — APPROVED** [review]

**Review by JoachimHenze — APPROVED** [review]
Approved now.

**Pull request merged**